app/src/main/java/org/oppia/android/app/help/thirdparty/ThirdPartyDependencyListViewModel.kt

```diff
@@ -0,0 +1,45 @@
+package org.oppia.android.app.help.thirdparty
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.android.R
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+/** View model in [ThirdPartyDependencyListFragment]. */
+class ThirdPartyDependencyListViewModel @Inject constructor(
+  val activity: AppCompatActivity
+) : ObservableViewModel() {
+  private val arrayList = ArrayList<ThirdPartyDependencyItemViewModel>()
+
+  /** Stores the list of third-party dependencies. */
+  val thirdPartyDependencyItemList: List<ThirdPartyDependencyItemViewModel> by lazy {
+    getRecyclerViewItemList()
+  }
+
+  private fun getRecyclerViewItemList(): ArrayList<ThirdPartyDependencyItemViewModel> {
+    val thirdPartyDependencyNames: Array<String> =
+      activity.resources.getStringArray(R.array.third_party_dependency_names_array)
+    val thirdPartyDependencyVersions: Array<String> =
+      activity.resources.getStringArray(
+        R.array.third_party_dependency_versions_array
+      )
+    thirdPartyDependencyNames.forEachIndexed { index, name ->
+      val thirdPartyDependencyItemViewModel =
+        ThirdPartyDependencyItemViewModel(
+          activity = activity,
+          name = omitVersion(name),
```

Comment:
Is this meant to be the human readable name or something we use to retrieve the license text? If the latter, could we not instead generate the XML files such that these two references are properly linked without needing to do any transformations?

Reply:
Removed the function and will change the script to contain dependency names without any version at the end.